src/main/kotlin/org/rust/cargo/runconfig/CargoCommandConfiguration.kt

```diff
@@ -25,6 +25,7 @@ class CargoCommandConfiguration(project: Project,
 
     var command: String = "run"
     var additionalArguments: String = ""
+    var environmentVariables: Map<String, String>? = null
```

Comment:
Can we use `var environmentVariables: Map<String, String> = emptyMap()` here and elsewhere to avoid dealing with `null`?